.github/workflows/release.yml

```
@@ -16,31 +16,35 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
           node-version: '16'
-          cache: yarn
+          cache: 'pnpm'
           registry-url: https://registry.npmjs.org
 
-      - name: yarn install
-        run: yarn install --frozen-lockfile
+      - name: pnpm install
+        run: pnpm install --frozen-lockfile
 
-      - name: yarn test
-        run: yarn test
+      - name: pnpm test
+        run: pnpm test
 
       - name: Deploy to Heroku
         env:
           HEROKU_API_TOKEN: ${{ secrets.HEROKU_API_TOKEN }}
           HEROKU_APP_NAME: release-drafter
         run: |
-          git fetch origin master
-          git push https://heroku:$HEROKU_API_TOKEN@git.heroku.com/$HEROKU_APP_NAME.git origin/master:master --force
+          git fetch origin main
+          git push https://heroku:$HEROKU_API_TOKEN@git.heroku.com/$HEROKU_APP_NAME.git origin/main:master --force
```

```suggestion
          git push https://heroku:$HEROKU_API_TOKEN@git.heroku.com/$HEROKU_APP_NAME.git origin/main:main --force
```
(not sure about this one)

---

I don't want to change it on heroku just yet, sounds scary to run `heroku repo:reset -a APP-NAME`